inspec/a2-iam-v2-no-legacy-policies/controls/get_authorized_projects.rb

```diff
@@ -8,10 +8,6 @@
   title 'getting all allowed projects for a user on v2.1'
 
  describe 'allowed projects' do
-
-    UNASSIGNED_PROJECT_ID = '(unassigned)'
-    UNASSIGNED_PROJECT_NAME = UNASSIGNED_PROJECT_ID
```

Comment:
I see you have moved these two constants to constants.rb just above... Then I'm puzzled: those constants are specific to this one test...?  Because I notice you define them again in role_access.rb...

Reply:
They are per-folder atm. I was just trying to get the output to stop complaining about constants being redefined (there were a lot of complaints).